gradle.properties

```diff
@@ -1,23 +1,23 @@
 org.gradle.caching=true
 puppycrawlCheckstyleVersion=8.18
 group=org.ballerinalang
-version=1.0.5-SNAPSHOT
-ballerinaLangVersion=2.0.0-alpha2-SNAPSHOT
+version=1.0.6-SNAPSHOT
```

Comment:
```suggestion
version=1.0.5

Reply:
`1.0.5` is the released version right? Why are we changing this back to the released version?

Reply:
We use this module version in ballerina-lang. The current master has 1.0.6 version [1]. 2.0.0-alpha2 doesn't. It has 1.0.5
[1] https://github.com/ballerina-platform/ballerina-lang/blob/master/compiler/ballerina-lang/src/main/java/org/ballerinalang/util/BLangCompilerConstants.java#L59